releasenotes/notes/graphql-add-span-events-errors-a0fb733673ff1f58.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    graphql: Add span events on GraphQL error spans.
```

Comment:
Can we add a few more words to this release note (one sentence tops) describing the value of this feature. For example we now capture more than one error.